pkg/api/testspec.proto

```diff
@@ -0,0 +1,46 @@
+syntax = 'proto3';
+
+package api;
+
+import "google/protobuf/duration.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "pkg/api/submit.proto";
+import "pkg/api/event.proto";
+
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) = true;
+
+// Defines a test case for the Armada test suite.
+// Defined as a proto message to enable unmarshalling oneof fields.
+message TestSpec {
+	// Jobs to submit.
```

Comment:
The whitespace on this proto file is not consistent.

Reply:
How so?

Reply:
Maybe I'm looking at it wrong but I agree with Kevin here.  At least t he way github is rendering it, some of the fields seems to be indented an extra tab